driver-sync/src/main/com/mongodb/client/DistinctIterable.java

```diff
@@ -92,6 +92,24 @@ public interface DistinctIterable<TResult> extends MongoIterable<TResult> {
      */
     DistinctIterable<TResult> comment(@Nullable BsonValue comment);
 
+    /**
+     * Sets the hint for which index to use. A null value means no hint is set.
+     *
+     * @param hint the hint
+     * @return this
+     * @since 5.3
+     */
+    DistinctIterable<TResult> hint(@Nullable Bson hint);
+
+    /**
+     * Sets the hint to apply.
+     *
+     * @param hint the name of the index which should be used for the operation
+     * @return this
+     * @since 5.3
+     */
+    DistinctIterable<TResult> hintString(@Nullable String hint);
```

Comment:
again, it would be great to add the following detail above:
```
 * <p>Note: If {@link DistinctIterable#hint(Bson)} is set that will be used instead of any hint string.</p>
 ```